guides/keras_nlp/quick_tour.py

```diff
@@ -0,0 +1,610 @@
+"""
+Title: KerasNLP quick tour
+Author: [jbischof](https://github.com/jbischof)
+Date created: 2022-12-15
+Last modified: 2022-12-15
+Description: An introduction to the KerasNLP API.
+"""
+"""
+<a
+href="https://colab.research.google.com/github/jbischof/keras-io/blob/quickstart/guides/ke
+ras_nlp/keras_nlp_quick_tour.ipynb" target="_parent"><img
+src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
+"""
+
+import keras_nlp
+import tensorflow as tf
+from tensorflow import keras
+import tensorflow_datasets as tfds
+
+# Use mixed precision for optimal performance
+keras.mixed_precision.set_global_policy("mixed_float16")
+
+"""
+# KerasNLP: Modular NLP Workflows for Keras
```

Comment:
This title can be removed or we get a double title with line 2.

Reply:
To me it looks kinda odd as rendered right now. Both this (which now renders as a section heading), and the top actual header read like titles for the whole guide. I would probably just rename this "Introduction", which would fit with [other guides](https://keras.io/guides/distributed_training/). Could also just ditch the second "KerasNLP", but "Introduction" would read clearer to me.

![image](https://user-images.githubusercontent.com/1389937/208852546-e37b761c-c628-47db-9c8f-b8ec103cf1c6.png)